tools/utils.py

```diff
@@ -38,16 +39,23 @@ def get_service_port(service, target=None, namespace='assisted-installer'):
         port = reply[4].split(":")[0]
     return port.strip()
 
-def get_service_url(service: str, target: Optional[str] = None, domain: str = "", namespace: str = 'assisted-installer') -> str:
+
+def get_service_url(service: str, target: Optional[str] = None, domain: str = "", namespace: str = 'assisted-installer', disable_tls: bool = False) -> str:
     # TODO: delete once rename everything to assisted-installer
     if target == "oc-ingress":
         service_host = f"assisted-installer.{get_domain(domain)}"
-        service_port = "80"
+        return to_url(service_host, disable_tls)
     else:
         service_host = get_service_host(service, target, namespace=namespace)
         service_port = get_service_port(service, target, namespace=namespace)
+        return f'http://{service_host}:{service_port}'
```

Comment:
Why not call to_url anyway?
If target is not oc-ingress than disable_tls should be true and then we'll use http

Reply:
I created the function so that we can call it when HTTPS is supported in non-oc-ingress scenarios. Right now everything else is HTTP.

Reply:
but this function handles the case of TLS disabled as well, why not use it if its there?